README.md

```diff
@@ -5,6 +5,12 @@
   <img src="https://img.shields.io/npm/dt/@dropbox/ttvc"/>
   <img src="https://img.shields.io/bundlephobia/minzip/@dropbox/ttvc"/>
   <img src="https://img.shields.io/bundlephobia/min/@dropbox/ttvc"/>
+  <img src="https://img.shields.io/github/checks-status/dropbox/ttvc/main"/>
+  <img src="https://img.shields.io/github/checks-status/dropbox/ttvc/main"/>
```

Comment:
It looks like you've got this badge listed twice.  Were you trying to include something else, or should we just remove one?